src/benchmarklib/abstract_query_generator.cpp

```diff
@@ -2,6 +2,8 @@
 
 namespace opossum {
 
+std::string AbstractQueryGenerator::setup_queries() const { return ""; }
```

Comment:
Why did we change from `preparation_queries` to `setup_queries`?
Because this one is ambiguous. I would expect to be able to call `TPCHQueryGenerator::setup_queries()` and then my queries are set up. You could use `get_setup_queries`, but even then `get_prepare_queries` would make more sense because it returns PREPARE queries.

Reply:
> ("preparation_query"; I know its derived from PREPARE; but I didn't immediately associate it with "ah, this returns PREPARE statements")

`get_preparation_queries` maybe? @mrzzzrm ?

Reply:
if you have to.